.github/workflows/fedora_29.yml

```diff
@@ -0,0 +1,120 @@
+name: Fedora 29 (RHEL 8.4), Python 3.8
+on:
+  workflow_dispatch:
+  pull_request:
+  merge_group:
+  push:
+    branches:
+      # - master
+      - 'releases/**'
+
+concurrency:
+  # github.ref is not unique in post-commit
+  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-fedora-29
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  Smart_CI:
+    runs-on: ubuntu-latest
+    outputs:
+      affected_components: "${{ steps.smart_ci.outputs.affected_components }}"
+      changed_components: "${{ steps.smart_ci.outputs.changed_components }}"
+      skip_workflow: "${{ steps.smart_ci.outputs.skip_workflow }}"
+    steps:
+      - name: checkout action
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          sparse-checkout: .github/actions/smart-ci
+
+      - name: Get affected components
+        id: smart_ci
+        uses: ./.github/actions/smart-ci
+        with:
+          repository: ${{ github.repository }}
+          pr: ${{ github.event.number }}
+          commit_sha: ${{ github.sha }}
+          ref_name: ${{ github.ref_name }}
+          component_pattern: "category: (.*)"
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          skip_when_only_listed_labels_set: 'docs'
+          skip_when_only_listed_files_changed: '*.md,*.rst,*.png,*.jpg,*.svg,*/layer_tests_summary/*,*/conformance/*'
+
+      - name: Show affected components
+        run: |
+          echo "${{ toJSON(steps.smart_ci.outputs.affected_components) }}"
+        shell: bash
+
+  Docker:
+    needs: Smart_CI
+    runs-on: aks-linux-4-cores-16gb-docker-build
+    container:
+      image: openvinogithubactions.azurecr.io/docker_build:0.2
+      volumes:
+        - /mount:/mount
+    outputs:
+      images: "${{ steps.handle_docker.outputs.images }}"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      - uses: ./.github/actions/handle_docker
+        id: handle_docker
+        with:
+          images: |
+            ov_build/fedora_29
+          registry: 'openvinogithubactions.azurecr.io'
+          dockerfiles_root_dir: '.github/dockerfiles'
+          changed_components: ${{ needs.smart_ci.outputs.changed_components }}
+
+  Build:
+    needs: [Docker, Smart_CI]
+    if: "!needs.smart_ci.outputs.skip_workflow"
+    uses: ./.github/workflows/job_build_linux.yml
+    with:
+      runner: 'aks-linux-16-cores-32gb'
+      container: '{"image": "${{ fromJSON(needs.docker.outputs.images).ov_build.fedora_29 }}", "volumes": ["/mount:/mount"], "options": "-e SCCACHE_AZURE_BLOB_CONTAINER -e SCCACHE_AZURE_CONNECTION_STRING"}'
+      affected-components: ${{ needs.smart_ci.outputs.affected_components }}
+      event-name: ${{ github.event_name }}
+      os: 'fedora_29'
+      build-contrib: false
+      build-debian-packages: false
+      build-js: false
+      build-rpm-packages: false
```

Comment:
I think we need to build RPMs as later we will release them as OpenVINO artifacts

Reply:
Do we also need to test them as it is done in Fedora 33?

I could drop Fedora 33 wf and move its RPM test job into Fedora 29.

Reply:
yes, let's replace 33 by 29 and keep test scope the same

Reply:
I tried running the RPM package test and got the following: `nothing provides python3-openvino-2024.5.0.11-1 = 2024.5.0.11-1 needed by openvino-2024.5.0-2024.5.0.11-1.noarch
 Problem 2: conflicting requests` in https://github.com/openvinotoolkit/openvino/actions/runs/10735993676/job/29775533820.
 Should we update the package to 2024? It is 2023 in the test job: 
![image](https://github.com/user-attachments/assets/ff79dca8-cecc-43ef-a00d-2ada437ef124)

Reply:
Seems like the `python3-openvino-2024.5.0.21-1` rpm is missing from the built RPM packages on Fedora 29. Could it be the Python version: 3.8 on Fedora 29 vs 3.9 on Fedora 33?

Reply:
I see multiple `sh: /usr/bin/python: No such file or directory` messages in the Build RPM packages step on Fedora 29, they were not present in the Fedora 33 pipeline.

Reply:
right, I saw the same..

Reply:
It did find Python: 
![image](https://github.com/user-attachments/assets/cd3ef58e-21e5-41b1-8ba1-66ecd662088f)
and completed the build process yet tried to use `/usr/bin/python3.8` for building the RPM packages themselves later:
![image](https://github.com/user-attachments/assets/778e7c31-9ce8-48ff-84a0-64cd95e0875e)
and complained about the `/usr/bin/python`:
![image](https://github.com/user-attachments/assets/00ba5b70-7555-4513-be99-29745ae21e95)
in: https://github.com/openvinotoolkit/openvino/actions/runs/10737772205/job/29780259018

Is this a bug or should we symlink `/usr/bin/python(3.8)` to the venv one?

I've provided the Python exec via `-DPython3_EXECUTABLE`.